.aliases.sh

```diff
@@ -61,6 +61,7 @@ alias grep='grep --color=auto' # Enable colorized grep output
 ###################################################################################################
 alias vi='vim' # Use Vim as default text editor
 alias q='exit' # Exit the current shell session
+alias :x=':wq!' #use to save changes in the text editor
```

Comment:
I don't think it's a good idea to use wq! (forceful) ? It would be safe and appreciated if it will be like 
 alias :x=':wq'  # Use to save changes in the text editor
 
 I would like to know your thoughts as well...